go.mod

```diff
@@ -1,8 +1,14 @@
 module github.com/fyne-io/image
 
-go 1.14
+go 1.18
```

Comment:
The upcoming version of Fyne is targeting 1.19 as baseline. You might as well bump it once more.

Reply:
Makes sense, will add a commit for that which you can squash with 5bdced54